sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSiteArchiveState

```diff
@@ -0,0 +1,70 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spositearchivestate
+applicable: SharePoint Online
+title: Set-SPOSiteArchiveState
+schema: 2.0.0
+author: aditi-gangwar
+ms.author: adigan
+ms.reviewer:
+---
+
+# Set-SPOSiteArchiveState
+
+## SYNOPSIS
+
+Sets the archive state of the file. Can be used to archive, reactivate sites.
```

Comment:
Is "file" the right term here or is it site?

Reply:
Thanks for pointing out. 
Edited.